test/resources/expectedOutputs/java-rdf4j/src/main/java/com/inrupt/testing/SCHEMA_INRUPT_EXT.java

```diff
@@ -55,23 +55,23 @@ private static String FULL_IRI(final String localName) {
     /**
      * A person (alive, dead, undead, or fictional).
      *
-     * This term has [4] labels and comments, in the languages [de, es, fr, it].
+     * This term provides multilingual descriptions, but has a mismatch between its labels and comments, with [6] labels in languages [NoLocale, de, en, es, fr, it], but [5] comments in languages [NoLocale, de, es, fr, it].
```

Comment:
I'm assuming this sort of thing is a schema.org issue?

Reply:
It's good that you pointed this out (I was aware of it, but kinda brushed it aside (as I understand why it's happening) - but it's ripe for causing other people confusion too).
So no, this comes from our sample extension to Schema.org ('cos Schema.org doesn't provide any translations at all today) - so it's our simply extension that is providing these translations (i.e., test/resources/vocabs/schema-inrupt-ext.ttl).
But in fact, this does illustrate an interesting edge case (caused by the frustrating difference between RDF string literals that have explicit language tags, and those that don't (e.g. `<> rdfs:label "My label"@en .` and `<> rdfs:label "My label" .`)).
In this particular case, the original Schema.org does provide a comment for `schema:Person` in English, but they don't provide an explicit language tag (which I actually consider 'bad form' from them, but it's very common in vocabs out there in the wild) - here is the official Schema.org Turtle (viewable from [here](https://schema.org/docs/developers.html)):

```turtle
schema:Person a rdfs:Class ;
    rdfs:label "Person" ;
    rdfs:comment "A person (alive, dead, undead, or fictional)." ;
    rdfs:subClassOf schema:Thing ;
    ns3:equivalentClass <http://xmlns.com/foaf/0.1/Person> ;
    schema:source <http://www.w3.org/wiki/WebSchemas/SchemaDotOrgSources#source_rNews> .
```

So our AG will look for comments with either "@en", or no language tag at all (and just assume that it's in English!). Therefore our extension doesn't need to duplicate the `rdfs:comment` from the original Schema.org definition. Although it could do that, and then stick an explicit "@en" on it's copy, which would then remove this apparent 'mismatch' (and arguably would be more correct (i.e., since it wouldn't rely on people looking for "@en", and if not found then having to fall back to looking for no language tag and if found _assuming_ that that value is actually in English)).
Or, I could just detect a mismatch that is based purely on a "@en" vs "NoLocale" difference, and remove the "but has a mismatch" wording. I've implemented that now: e7a5c94